boards/native/native_posix/doc/index.rst

```diff
@@ -11,18 +11,18 @@ Native POSIX execution (native_posix)
 Overview
 ********
 
-``native_posix`` is the predecessor of :ref:`native_sim<native_sim>`.
+``native_posix/native`` is the predecessor of :ref:`native_sim<native_sim>`.
 Just like with :ref:`native_sim<native_sim>` you can build your Zephyr application
 with the Zephyr kernel, creating a normal Linux executable with your host tooling,
 and can debug and instrument it like any other Linux program.
 
 But unlike with :ref:`native_sim<native_sim>` you are limited to only using the host C library.
-:ref:`native_sim<native_sim>` supports all ``native_posix`` use cases.
+:ref:`native_sim<native_sim>` supports all ``native_posix/native`` use cases.
```

Comment:
From an offline discussion with @nordicjm I understand the long term plan is to not allow anymore targeting boards without the soc (in tree, but users may still be able to when building directly(?)). I think this change will be too annoying, as to warrant a discussion in the TSC.